uswds/uswds
# USWDS - Remove @types/eslint from devDependencies

## Related issue
Closes https://github.com/uswds/uswds/issues/4856

## Problem statement
Adding `@types/eslint` as a devDependency was a temporary solution while `@types/eslint` was down due to a bad version release. 

We no longer need to pin this dependency, so this PR removes it from `package.json`.

## References
Original USWDS issue: [USWDS - Bug: Dependency failure @types/eslint](https://github.com/uswds/uswds/issues/4840) 
`@types/eslint` issue: [v8.4.4 missing from NPM registry](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/61032)

## Testing and review
1.  Run a fresh `npm install`
2. Check for successful install

Before you hit Submit, make sure you’ve done whichever of these applies to you:
